static/src/inline-svgs/icon/gifting.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="18" viewBox="0 0 16 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
has this been run through svgo? https://jakearchibald.github.io/svgomg/

Reply:
```xml
<svg width="16" height="18" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M4.5 4.3c1 0 1.8-.7 2.5-2L4 0h-.7l-.6 2.4.4.3c.7 1.6.5 1.6 1.4 1.6zm7 0c-1 0-1.8-.7-2.5-2L11.8 0h.8l.6 2.4.1.3c-.7 1.6-1 1.6-1.8 1.6z" fill="#fff"/><path d="M6.5 6.6H1V3.4l.5-.5h5v3.7zm0 2.7v7.6H2l-.5-.5V9.3h5zM15 6.6H9.5V2.9h5l.5.5v3.2zm-.5 2.7v7.1l-.5.5H9.5V9.3h5z" fill="#fff" stroke="#fff"/></svg>
```
![image](https://user-images.githubusercontent.com/7304387/102613312-2e878f80-412a-11eb-83c5-e73e9e9dd593.png)
![image](https://user-images.githubusercontent.com/7304387/102613357-3c3d1500-412a-11eb-80e2-4bbe99e9ff4c.png)